examples/ffi-struct/struct.c

```diff
@@ -0,0 +1,21 @@
+struct Inner
+{
+	int x;
```

Comment:
Can you try to use the same code style as in the other C source in this project?

Specifically, note that 2 spaces (not tabs) are used for indentation.